core/tests/build_sources/extensions/base.py

```diff
@@ -47,16 +47,55 @@
 from core.domain import object_registry
 from core.domain import visualization_registry
 from extensions import domain
+from extensions.visualizations import models
```

Comment:
This file doesn't need to be typed (but feel free to keep it typed).

Reply:
ok i'll keep that as it is